airflow/models.py

```diff
@@ -69,8 +69,6 @@
 
 Base = declarative_base()
 ID_LEN = 250
-SQL_ALCHEMY_CONN = configuration.get('core', 'SQL_ALCHEMY_CONN')
-DAGS_FOLDER = os.path.expanduser(configuration.get('core', 'DAGS_FOLDER'))
```

Comment:
Replace one-time retrieval of the setting by the configuration convenience method that checks the `DAGS_FOLDER` of the current configuration. (Important because the configuration will be changed during tests).